From: Shenghao Ding <shenghao-ding@ti.com>
Date: Thu, 18 Jul 2024 15:00:22 +0800
Subject: ASoC: tas2781: fix the license issue for tas781 firmware
To: linux-kernel@vger.kernel.org, linux-firmware@kernel.org
Origin: https://gitlab.com/kernel-firmware/linux-firmware/-/commit/b96c61091133a3c436c5be895c8c295673ba41f3

Fix the license issue for tas781 firmware, which are shipped as .bin
opaque. Remove the GPL-2+, which imposes having the source code
available.

Signed-off-by: Shenghao Ding <shenghao-ding@ti.com>
---
 WHENCE | 12 ++++--------
 1 file changed, 4 insertions(+), 8 deletions(-)

--- a/WHENCE
+++ b/WHENCE
@@ -7322,10 +7322,8 @@ Link: TAS2XXX38E0.bin -> ti/tas2781/TAS2
 Link: TIAS2781RCA2.bin -> ti/tas2781/TIAS2781RCA2.bin
 Link: TIAS2781RCA4.bin -> ti/tas2781/TIAS2781RCA4.bin
 
-Licence: Allegedly GPLv2+, but no source visible. Marked:
-	 Copyright (C) 2023 Texas Instruments
-
-Found in hex form in kernel source.
+Licence: Redistributable. See LICENCE.ti-tspa for details. Marked:
+	 Copyright (C) 2023 - 2024 Texas Instruments
 
 --------------------------------------------------------------------------
 
@@ -7335,10 +7333,8 @@ File: ti/tas2563/INT8866RCA2.bin
 Link: TAS2XXX3870.bin -> ti/tas2563/TAS2XXX3870.bin
 Link: INT8866RCA2.bin -> ti/tas2563/INT8866RCA2.bin
 
-Licence: Allegedly GPLv2+, but no source visible. Marked:
-	 Copyright (C) 2023 Texas Instruments
-
-Found in hex form in kernel source.
+Licence: Redistributable. LICENCE.ti-tspa for details. Marked:
+	 Copyright (C) 2023 - 2024 Texas Instruments
 
 --------------------------------------------------------------------------
 
